packages/manager/apps/pci-rancher/src/public/translations/pci-rancher/dashboard/Messages_fr_FR.json

```
@@ -30,7 +30,8 @@
   "createRancherInfoMessage": "Bien que le service soit gratuit durant la phase Beta, votre utilisation sera, par la suite, facturée à l’heure, en fonction du nombre total de vCPU de chacun des nœuds de travail (worker nodes) de vos « downstream clusters » Rancher, en considérant un <strong>minimum de 20 vCPU facturés.</strong>",
   "createRancherName": "Nom",
   "createRancherDescription": "Vous pourrez gérer plusieurs « downstream clusters » au sein de ce service.",
-  "createRancherPlaceholder": "NomDuRancher",
+  "createRancherPlaceholder": "mon_rancher",
```

```suggestion
  "createRancherPlaceholder": "mon rancher",
```

---

"mon rancher" is not accepted as a rancher name since the service name must be between 3 and 63 characters, must start and end with lower-case alphanumeric characters (a to z and 0 to 9), and may include the following punctuation marks  '_', '.' and '-'

---

Ok in that case, the placeholder shouldn't be translated and kept in english

```suggestion
  "createRancherPlaceholder": "rancher_name",
```